src/_data/archivePages.js

```diff
@@ -1,9 +1,36 @@
 const wrgConfig = require('../../getConfig')();
 
-module.exports = () =>
-  wrgConfig.wacz_urls.map((url) => {
-    return {
-      name: url,
-      url: `/archive/#/${encodeURIComponent(url)}`,
-    };
-  });
+/**
+ * Transform archives data to site page data
+ * @param {string|Object} data - WACZ data, can be URL string or { name: string; url: string }
+ * @returns {Object} archive
+ * @returns {string} archive.name - Name of archive
+ * @returns {string} archive.waczURL - URL to WACZ file
+ * @returns {string} archive.pathname - Site page path
+ */
+function mapToPage(data, idx) {
+  let name, waczURL;
+
+  if (typeof data === 'string') {
+    name = data;
```

Comment:
I guess one idea is to use the filename as the name, `https://example.com/path/filename.wacz` -> `filename` for autocomplete/search, but not sure if that makes sense yet.. (and there could be dupes).
Could try this later if needed.